app.js

```diff
@@ -1,15 +1,25 @@
+const persist = require('choo-persist')
 const mount = require('choo/mount')
 const log = require('choo-log')
+const xtend = require('xtend')
 const choo = require('choo')
 
-const mainView = require('./pages/main')
+const opts = {
+  filter: (state) => {
+    state = xtend(state)
```

Comment:
@karissa `Object.assign()` is JavaScript's own solution to this problem. It's part of the language's core and should therefore be preferred, especially in environments like Electron where you know exactly that you'll get an up to date version of V8.